packages/ember-css-modules/lib/htmlbars-plugin/index.js

```diff
@@ -62,15 +62,15 @@ module.exports = class ClassTransformPlugin {
   }
 
   transformStatement(node) {
-    if (node.path.original === 'local-class') {
+    if (!this.isNodePathLiteral(node.path) && node.path.original === 'local-class') {
```

Comment:
Shouldn't we fix this differently by using either `.original` or `.value`?

Reply:
Honestly it's been so long since I've looked at this issue, I can't tell you what I expected to see anymore, but suspect it was just the warning "spew" as you say in your attempt.